server/handlers.go

```diff
@@ -34,19 +34,9 @@ func (svr *Server) handleHealth(w http.ResponseWriter, _ *http.Request) error {
 // GET ROUTES
 // =================================================================================================
 
-// handleGetStdCommitment handles the GET request for std commitments.
-func (svr *Server) handleGetStdCommitment(w http.ResponseWriter, r *http.Request) error {
-	versionByte, err := parseVersionByte(w, r)
-	if err != nil {
-		return fmt.Errorf("error parsing version byte: %w", err)
-	}
-
-	commitmentMeta := commitments.CommitmentMeta{
-		Mode:     commitments.OptimismGeneric,
-		Version:  commitments.EigenDACommitmentType(versionByte),
-		Encoding: commitments.RLPEncoding,
-	}
-
+// handleGetEigenDA helper function used for processing gets for an eigenda backend type
+func (svr *Server) handleGetEigenDA(w http.ResponseWriter, r *http.Request,
```

Comment:
dont think this should be called handle because its not a routing table entrypoint

Reply:
or maybe prefix name with a `_`?